src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -11,6 +11,7 @@
 
 namespace ApiPlatform\Core\Bridge\Symfony\Bundle\DependencyInjection;
 
+use Doctrine\ORM\Version;
```

Comment:
Really not a fan of unqualified class/interface names like this. `Version` is not descriptive at all. (Version of what?)